dpos/network.go

```diff
@@ -313,9 +313,8 @@ func NewDposNetwork(account account.Account,
 	copy(pid[:], account.PublicKeyBytes())
 	server, err := p2p.NewServer(&p2p.Config{
 		PID:              pid,
+		EnableHub:        true,
```

Comment:
If this parameter is configurable, it should be taken from the config.json file; if not, it should not appear in the config.

Reply:
This is not the config of the config file, this config is the parameters used to initiate a P2P server instance.